db/seeds.rb

```diff
@@ -62,17 +62,30 @@
 
 
 # Okay, now let's seed a few bookings with reviews
+puts "Seeding bookings and reviews..."
+200.times do
 
-100.times do
+  m = rand(1..7)
+  sd = rand(1..15)
+  ed = sd + rand(1..15)
+  st = Date.new(2020,m,sd)
+  ed = Date.new(2020,m,ed)
+
+  user_this_booking = users.sample.id
```

Comment:
wouldn't use the id but the user directly, id's might change if you reseed your file without dropping the database. instead go for users.sample and then down below in the booking, user: user_this_booking